### ISSUE TITLE
Accessibility: Widget titles should be HTML headers

### ISSUE THREAD

**Issue by mnapoli** [piwik/piwik#7005]
refs #4167

Widget titles should be HTML headers (`h2` or `h3`?) in order to facilitate the navigation in the dashboard using a screen reader. Currently there is no header on the home page.

**Comment by mnapoli** [comment]
Was fixed in #7015
